### DISCUSSION TITLE
希望能保留搜索栏原版的搜索联想结果

### DISCUSSION THREAD

**Discussion by DonutTrump** [the1812/Bilibili-Evolved#2677] [功能建议 / Ideas]
感谢各位大佬的辛苦开发，从1.12升级到了2.0.7,体验很好。但是发现更新后的搜索栏在搜索时联想结果变弱了很多，希望能保留原版的搜索联想结果，即用拼音缩写字母来快速搜索内容，比如搜“xyh”就能联想出“小约翰可汗”。感觉这个非常实用，希望能够保留。最后再次感谢各位开发者

**Comment by the1812** [comment]
联想好像有点 bug, 本来应该能搜出来的

**Reply by DonutTrump** [reply]
好像是有点小问题，但是在b站搜索页(search.bilibili.com)的主搜索栏，这个搜索联想功能没有问题，可以用拼音字母缩写联想出结果来，只是页面顶端的搜索栏不支持了

**Reply by Meltryllis163** [reply]
这个功能现在似乎还是不存在（或者依然存在Bug？），对于输入法云不到的词汇（比如新游戏名称、UP主名称等等），这个功能感觉还是很好用的，没有的话有点难受。
